dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentTypeLoader.java

```diff
@@ -0,0 +1,303 @@
+                    .append(field.variable())
+                    .append("ImageHeight =\"")
+                    .append(150)
+                    .append("\" )"); // Originak value was 65
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageExtension =\"gif\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageURI =\"")
+                    .append(uri)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageTitle =\"[ Test Image Structure ]\" )");
+            } else if (field instanceof FileField) {
+                // Identifier id = (Identifier) InodeFactory.getChildOfClassByRelationType(content,
+                // Identifier.class, field.getFieldRelationType());
+                String uri = "/html/images/shim.gif";
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("FileInode =\"")
+                    .append(Long.MAX_VALUE)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("FileExtension =\"gif\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("FileURI =\"")
+                    .append(uri)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("FileTitle =\"[ Test File Structure ]\" )");
+            } else if (field instanceof SelectField) {
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("SelectLabelsValues = \"")
+                    .append(fv.replaceAll("\\r\\n", " ")
+                        .replaceAll("\\n", " "))
+                    .append("\")");
+            } else if (field instanceof RadioField) {
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("RadioLabelsValues = \"")
+                    .append(fv.replaceAll("\\r\\n", " ")
+                        .replaceAll("\\n", " "))
+                    .append("\" )");
+            } else if (field instanceof CheckboxField) {
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("CheckboxLabelsValues = \"")
+                    .append(fv.replaceAll("\\r\\n", " ")
+                        .replaceAll("\\n", " "))
+                    .append("\" )");
+            } else if (field instanceof DateField) {
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append(" =\"[ ")
+                    .append(field.variable())
+                    .append(" ]\")");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " ]\")" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)